### DISCUSSION TITLE
about the turn server

### DISCUSSION THREAD

**Discussion by abdeldjalil-fellah** [Ylianst/MeshCentral#5432] [General]
- which port is used by the built-in turn server? or where is the socket file located? 
- how to use a custom protected turn server with url:port and user/password?

**Comment by si458** [comment]
are you talking about the STUN webrtc settings?
The STUN servers used for WebRTC if not specified are Google and Mozilla servers, meshcentral doesnt have stun server built in?

**Reply by abdeldjalil-fellah** [reply]
I thought it has https://meshcentral1.rssing.com/chan-8632689/article91.html
can I specify the user/password for the STUN server in config.json?

**Reply by si458** [reply]
do you mean a user/pass to connect to an external stun server?
i dont believe this is possible?
https://github.com/Ylianst/MeshCentral/blob/master/meshcentral-config-schema.json#L748-L773

**Reply by abdeldjalil-fellah** [reply]
I think this is a missing option
